src/govuk/components/accordion/accordion.js

```diff
@@ -104,6 +119,18 @@ Accordion.prototype.initHeaderAttributes = function ($headerWrapper, index) {
   $button.setAttribute('id', this.moduleId + '-heading-' + (index + 1))
   $button.setAttribute('aria-controls', this.moduleId + '-content-' + (index + 1))
 
+  // Create show / hide icons with text.
+  var $showIcons = document.createElement('span')
+  $showIcons.classList.add(this.sectionShowHideToggleClass)
+
+  // Build additional wrapper for toggle text, place icon before wrapped text.
+  var $wrapperShowHideIcon = document.createElement('span')
+  var $icon = document.createElement('span')
+  $icon.classList.add(this.upChevronIconClass)
+  $showIcons.appendChild($icon)
+  $wrapperShowHideIcon.classList.add(this.sectionShowHideTextClass)
```

Comment:
Only a small thing but the variable name is slightly confusing as it contains the toggle text.

Reply:
Updated the naming to be clearer to `wrapperToggleText`